pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -116,22 +110,36 @@ ENV REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases/download/${MAMBA_VERSION}/Mambaforge-${MAMBA_VERSION}-Linux-x86_64.sh \
  && chmod +x ~/mambaforge.sh \
  && ~/mambaforge.sh -b -p /opt/conda \
- && rm ~/mambaforge.sh \
- && /opt/conda/bin/mamba update -y conda \
- && /opt/conda/bin/mamba install -y -c conda-forge \
+ && rm ~/mambaforge.sh 
+
+# install core conda dependency packages (and pin essential packages)
+RUN /opt/conda/bin/mamba install -y -c conda-forge \
+    conda=23.1.0 \
+    "cryptography>39.0.0" \
+    cmake \
+    requests \
+    pyopenssl \
     python=$PYTHON_VERSION \
     cython \
-    mkl \
-    mkl-include \
-    parso \
-    typing \
-    h5py \
     # force conda to install h5py with openmpi build, not mpich (rubik error)
     # https://anaconda.org/conda-forge/h5py/files?page=2
     "mpi4py>=3.1.4,<3.2" \
     mpi4py \
-    requests \
-    pyopenssl \
+    h5py 
+
+# Install AWS-PyTorch
+RUN /opt/conda/bin/mamba install -y pytorch=${PYTORCH_VERSION} \
+  torchvision torchaudio torchtext=0.15.0 cpuonly \
```

Comment:
The conda channel provided `torchtext=0.15.1`. To see a list of package provided in the conda channel. Run `conda search --override -c https://aws-ml-conda-pre-prod-ec2.s3.us-west-2.amazonaws.com/`

Reply:
Just spotted the error, fixed now!